app/views/posts/index.html.erb

```diff
@@ -1,11 +1,15 @@
-<%= stylesheet_link_tag "views" %>
+
 <!DOCTYPE html>
 <html>
 <head>
 	<meta charset="utf-8">
 	<title>user's posts</title>
 </head>
 <body>
-  <h1>Here is a list of posts for a user</h1> 
+<main id ="container"> 
+  <%= render partial:'users/user', locals:{user: @user} %>
```

Comment:
Please use `#recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb`.